src/core/execute.c

```diff
@@ -731,21 +731,21 @@ static ConfirmAnswer ask_for_confirmation(const char *vc, const char *cmdline) {
                 return CONFIRM_EXECUTE;
         }
 
-        r = ask_char(&c, "yns", "Execute %s? [Yes, No, Skip] ", cmdline);
+        r = ask_char(&c, "yfs", "Execute %s? [Yes, Fail, Skip] ", cmdline);
```

Comment:
i still don't like that this is changed with no compat. The least you should add is a NEWS entry about this, explaining this.